website/frontend/templates/docs/build-windows.html

```diff
@@ -125,30 +204,98 @@ <h3 id="nsis">NSIS 2.46</h3>
+        
+      <p>
+        <b>Note:</b> There is also apparently a cut-down compiler specifically for compiling Python 2.7.x modules available at 
+        <a rel="nofollow" target="_blank" href="https://www.microsoft.com/en-gb/download/details.aspx?id=44266">https://www.microsoft.com/en-gb/download/details.aspx?id=44266</a>,
+        though the author has not tried this.
+      </p>
+
+      <h3 id="visual_cpp">Visual C++</h3>
+      
+      <p>
+        Before you can build the Python executable and installer, you need to build the astrcmp C extension and 
+        language internationalisation. 
+        Start the VS console using "Visual Studio 2008 Command Prompt" or "Visual C++ 2015 MSBuild Command Prompt", 
+        go to the Picard source directory and run these commands:
+      </p>
+
+      <pre>python setup.py build_ext -i
+python setup.py build_locales -i</pre>
+
+      <p>
+        Then you need to copy the following files from Visual Studio to 
+        to the source code directory, so that py2exe can find them:
+      </p>
+        
+      <ul>
+        <li>
+          Picard v2 / Python 3 - msvcp140.dll from 
+          <i>C:\Program&nbsp;Files&nbsp;(x86)\Microsoft&nbsp;Visual&nbsp;Studio&nbsp;14.0\VC\redist\x86\Microsoft.VC140.CRT</i>
+        </li>
+        <li>
+          Picard v1 / Python 2 - msvcr90.dll and msvcp90.dll from 
+          <i>C:\Program&nbsp;Files&nbsp;(x86)\Microsoft&nbsp;Visual&nbsp;Studio&nbsp;9.0\VC\redist\x86\Microsoft.VC90.CRT</i>
+        </li>
+      </ul>
+        
+      <p>
+        You also need copy discid.dll (libdiscid) to the picard source.
+      </p>
+        
       <p>
         Then you can build the installer using this command:
       </p>
-      <pre>
-        python setup.py bdist_nsis</pre>
+      <pre>python setup.py bdist_nsis</pre>
 
       <p>
-        The setup is installer\picard-setup-1.x.exe and an executable for your system is dist\picard.exe, which
+        The setup is installer\picard-setup-x.x.exe and an executable for your system is dist\picard.exe, which
         you can use to create a desktop shortcut.
       </p>
 
       <h1 id="known_issues">Known Issues</h1>
       <p>
-        It is possible that the resulting picard.exe does not show the proper file icon. This is probably a bug
-        of py2exe on Vista. You can fix the icon with the tool
+        It is possible that the resulting picard.exe does not show the proper file icon.
+        This is probably a bug of py2exe on Vista. 
+        You can fix the icon with the tool
         <a rel="nofollow" href="http://angusj.com/resourcehacker/">Resource Hacker</a> or any other tool, that
         can edit the resources in executables.
       </p>
 
+      <h1 id="support">Support</h1>
+      <p>
+        If these instructions do not work, please research and try to solve the problem yourself - 
+        and report the issue and answer in IRC freenode/#metabrainz or by submitting a PR against Picard-Website.
```

Comment:
Please do not report issues on IRC, use JIRA for that.

Also, please make Picard-Website a link to the repository.